docs/source/pipeline.rst

```diff
@@ -0,0 +1,270 @@
+scientists’ machine learning pipelines. They are built on a
+human-readable data format: **YAML**.
+
+==============================
+Setting Up Your First Pipeline
+==============================
+
+Requirements
+~~~~~~~~~~~~
+| To begin, you must:
+| - Have access to a machine that has Vertica installed
+| - Install Python on your machine
+| - Install Verticapy
+
+Create Your First YAML files
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+   | **THE BASIC RULES OF ANY YAML FILE:**
+   | - YAML is case sensitive 
+   | - The files should have **.yaml** as the extension 
+   | - YAML does not allow the use of tabs while creating YAML files
+
+| The information in connection.yaml will be the same you use in Verticapy.
+| ``$ vim connection.yaml``
+
+.. code:: yaml
+
+   # Replace with your information #
+   host: "127.0.0.1"
+   port: "5433"
+   database: ""
+   password: ""
+   user: "dbadmin"
+
+| The information in your pipeline yaml file will outline the flow of data.
+| ``$ vim gettingStarted.yaml``
+
+.. code:: yaml
+
+   schema: "public"
+   pipeline: "gettingStarted"
+   table: public.example_table
+   # steps:
+       # ingest: 
+       # transform: 
+       # train:
+       # test:
+
+Run the Parser
+~~~~~~~~~~~~~~
+
+| The parser will follow the general format:
+| ``$python -m verticapy.pipeline.parser [connection_file] [input_file] -o [output file]``
+
+| Both of the following will generate a sql file:
+  **gettingStarted.sql**
+| ``$python -m verticapy.pipeline.parser connection.yaml gettingStarted.yaml``
+| ``$python -m verticapy.pipeline.parser connection.yaml gettingStarted.yaml -o gettingStarted.sql``
+
+Dropping the Pipeline
+~~~~~~~~~~~~~~~~~~~~~
+
+| If you are done with the pipeline and want to drop all ingestions,
+  views, models, or stored procedures associated with it, you can do
+  either of the following:
+| **In a VSQL terminal:**
+| ``verticadb21455=> CALL drop_pipeline([schema name], [pipeline name]);``
+| **In VerticaPy cell**
+
+.. code:: sql
+
+   %%sql
+   CALL drop_pipeline([schema name], [pipeline name]);
+
+| for the example above running the sql would drop the pipeline:
+| ``CALL drop_pipeline('public', 'gettingStarted');``
+
+Now you should be ready to quickly build new pipelines!
+
+=============
+Documentation
+=============
+
+Ingestion
+~~~~~~~~~
+
+| For more information on how to customize this step: `DataLoader <https://docs.vertica.com/23.4.x/en/sql-reference/statements/create-statements/create-data-loader/#arguments>`__ and `COPY <https://docs.vertica.com/23.4.x/en/sql-reference/statements/copy/>`__
+
+.. code:: yaml
+
+     ingest:
+         from: '~/data/bucket/*'
+         delimiter: ','
+         retry_limit: 'NONE'
+         retention_interval: "'15 days'"
+
+Transform
+~~~~~~~~~
+
+| `transform functions documentation <https://www.vertica.com/python/documentation/1.0.x/html/vdataframe.html#features-engineering>`__
+
+| ``public.winequality``
+.. image:: _static/winequal_table.png
```

Comment:
Instead of inputting static images which can be outdated, we should generate HTML images and use them. 

Below is the snippet from verticapy/datasets/loaders.py to show the table. You can adjust it to show your table:

``` 

    .. ipython:: python
        :suppress:

        from verticapy.datasets import load_market
        html_file = open("SPHINX_DIRECTORY/figures/datasets_loaders_load_market.html", "w")
        html_file.write(load_market()._repr_html_())
        html_file.close()

    .. raw:: html
        :file: SPHINX_DIRECTORY/figures/datasets_loaders_load_market.html
        
```

Instead of ``from verticapy.datasets import load_market``, you can use ``from verticapy.datasets import load_winequality``.
Additionally you can even select a subset of columns if you want:

``load_winequality()["density","fixed_acidity","pH","color"]``